src/pug/about.pug

```diff
@@ -34,8 +34,8 @@ block page
     div.box
       h2 Do you sell controllers?
       p Short answer: No.
-      p Long answer: Becoming a manufacturer and/or online shop requires a lot of work, money investment, certifications, legal preparations, and it is possibly much more boring that what we do now.
-      p So #[em maybe] in the future we could offer controller parts to buy directly from Input Labs website, though it depends on many factors.
+      p Long answer: We distribute some #[em DIY kits] to make easier building a controller, since not everyone has a 3D printer or soldering material.
+      p You can check each device #[+a('alpakka/components') components page] for places in which you can buy components and DIY kits.
```

Comment:
Please change also the next sentence to something like "the components page outlines where you can buy components or DIY kits".